components/trace_metric/src/metric.rs

```diff
@@ -2,10 +2,16 @@
 
 use std::{fmt, time::Duration};
 
+#[derive(Clone)]
+pub enum MetricOp {
+    Add,
+}
```

Comment:
How about renaming it as:
```suggestion
pub enum MetricAggregator {
    Sum,
}
```